content/docs/pipelines/tutorials/build-pipeline.md

```diff
@@ -125,6 +125,8 @@ The following notebooks are available:
 
 ## Next steps
 
+* Understand the various ways to use the [Kubeflow Pipelines 
```

Comment:
```suggestion
* Learn the various ways to use the [Kubeflow Pipelines 
```
Learn might be more accurate unless you had a specific reason to select "understand"